cxx/gendb_test.cc

```diff
@@ -973,4 +958,11 @@ BOOST_AUTO_TEST_CASE(test_unincorporate_reincorporate_new) {
   //            tt::tolerance(1e-3));
 }
 
+BOOST_AUTO_TEST_CASE(test_transition_reference_class) {
+  std::mt19937 prng;
+  GenDB gendb(&prng, schema);
+  setup_gendb(&prng, gendb, 20);
+  gendb.transition_reference_class_and_ancestors(&prng, "Record");
+}
```

Comment:
Please add one or more BOOST_TEST's as to the state of the gendb at this point.

Reply:
Done.